devicon.json

```
@@ -5746,6 +5746,31 @@
         "color": "#8dc63f",
         "aliases": []
     },
+    {
+        "name": "miro",
+        "altnames": [
+            "RealtimeBoard"
+        ],
+        "tags": [
+            "software",
+            "whiteboarding-platform",
+            "project-management"
```

The project seems to be open-source as well (link: <https://github.com/miroapp>).}
```suggestion
            "project-management",
            "open-source"
```